### ISSUE TITLE
Open Source

### ISSUE THREAD

**Issue by freenerd** [mapbox/price-pigeon#1]
@KaiBot3000 can this be open source?

**Comment by KaiBot3000** [comment]
@freenerd that's my goal!

**Comment by KaiBot3000** [comment]
I'm giving this another shot!

Here's my checklist:
- [x] review history for sensitive info like tokens, customer names, etc.
- [x] review history for sensitive URLs like staging servers
- [x] review open and closed issues for sensitive info
- [x] add templates for PR+Issues
- [x] add license
- [x] get final check-off from @mapbox/security

**Comment by KaiBot3000** [comment]
I looked over every single commit 😅  and confirmed there isn't any sensitive info in the commit history. There are a few links/references to private projects in issues. I added a `@mapbox` to the links to make it clear that they're internal projects instead of deleting them, so that Mapbox folks could still get that context.

I also added a bunch of things like PR templates in https://github.com/mapbox/price-pigeon/pull/13

This is ready for review!
